masonry/src/widgets/grid.rs

```diff
@@ -204,31 +218,35 @@ impl Grid {
         this.ctx.request_layout();
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Set the spacing between grid items.
     pub fn set_spacing(this: &mut WidgetMut<'_, Self>, spacing: f64) {
         this.widget.grid_spacing = spacing;
         this.ctx.request_layout();
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Set the number of columns of the grid.
     pub fn set_width(this: &mut WidgetMut<'_, Self>, width: i32) {
         this.widget.grid_width = width;
         this.ctx.request_layout();
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Set the number of rows of the grid.
     pub fn set_height(this: &mut WidgetMut<'_, Self>, height: i32) {
         this.widget.grid_height = height;
         this.ctx.request_layout();
     }
 
-    #[expect(missing_docs, reason = "TODO")]
+    /// Get a mutable reference to the child at `idx`.
+    ///
+    /// # Panics
+    ///
+    /// Panics if `idx` is out of bounds.
     pub fn child_mut<'t>(
         this: &'t mut WidgetMut<'_, Self>,
         idx: usize,
-    ) -> Option<WidgetMut<'t, dyn Widget>> {
-        let child = this.widget.children[idx].widget_mut()?;
-        Some(this.ctx.get_mut(child))
+    ) -> WidgetMut<'t, dyn Widget> {
```

Comment:
Confusing to be changing API in the middle of documentation changes.

Reply:
I stand by it. The behavior doesn't change, we were always returning `Some`, the only consumer was unwrapping it, etc.

I could technically split it out into a second PR and add a TODO in this PR or something, but I don't see a benefit.

Reply:
I think it not being called out proactively is concerning, as this is quite a small part of a big PR.
That is, I agree that changing it here is fine; doing the documentation revealed the absurdity of the `widget_mut` method, and following it through led to this change naturally.

Fwiw, one possibility is that the intention from @jaredoconnell was that the grid would support spacers of some kind (maybe grid lines?). But I can't really see how that would be meaningful with this API.